README.md

```diff
@@ -158,34 +158,82 @@ An export configuration entry in the `export` arrays contains the following keys
-*  `QUEUE_POLL_INTERVAL`: the queue is polled every minute by default.
-*  `WAIT_FOR_INITIAL_SYNC`: wait for initial sync. Defaults to 'true', mainly meant to disable for debugging purposes
-* `MU_CALL_SCOPE_ID_PUBLICATION_GRAPH_MAINTENANCE (default: http://redpencil.data.gift/id/concept/muScope/deltas/publicationGraphMaintenance)`: can be configured to work with scopes in delta-notifier. This is fired when a updates are performed on the publication graph. Most of the services in your stack wouldn't be interested by this update, so best to add this in the deltanotifier configuration as scope to exclude. So we reduce load on the system, and potential confusion.
-* `MU_CALL_SCOPE_ID_INITIAL_SYNC (default: http://redpencil.data.gift/id/concept/muScope/deltas/initialSync)` : can be configured to work with scopes in delta-notifier. This when the publication graph is initially synced. Most of the services in your stack (if not all) wouldn't be interested by this update, so best to add this in the deltanotifier configuration as scope to exclude.
-* `HEALING_PATCH_GRAPH_BATCH_SIZE: Size of insert/delete batches. Defaults to 100 triples`
+* `USE_VIRTUOSO_FOR_EXPENSIVE_SELECTS (default: false)`: Whether to use virtuoso for expensive selects.
 * `SKIP_MU_AUTH_INITIAL_SYNC: Initial sync is expensive. Mu-auth can be skipped here. But be aware about what this implies!`
-* `UPDATE_PUBLICATION_GRAPH_SLEEP (default: 1000): Sleep between batch updates in the update of the publication graph`
+* `VIRTUOSO_ENDPOINT (default: http://virtuoso:8890/sparql)`: Location of the virtuoso endpoint.
+* `MU_AUTH_ENDPOINT (default: http://database:8890/sparql)`: Location of the mu-auth endpoint
+* `PUBLICATION_VIRTUOSO_ENDPOINT (default: VIRTUOSO_ENDPOINT)`: Location of the virtuoso endpoint.
+* `PUBLICATION_MU_AUTH_ENDPOINT (default: MU_AUTH_ENDPOINT)`: Location of the mu-auth endpoint
+* `SERVE_DELTA_FILES (default: false)`: Whether to serve delta files.
+* `LOG_OUTGOING_DELTA (default: false)`: Whether to log the delta messages as sent
+* `DELTA_INTERVAL (default: 1000)`: Interval between outgoing delta messages
+* `PRETTY_PRINT_DIFF_JSON (default: true)`: Whether to pretty print the diff json
+* `ERROR_GRAPH (default: http://mu.semte.ch/graphs/system/errors)`: The graph in which the errors should be stored.
+* `RELATIVE_FILE_PATH (default: "deltas")`: relative path of the delta files compared to the root folder of the file service that will host the files.
+* `FILES_GRAPH (default: http://mu.semte.ch/graphs/public)`: The graph in which the files should be stored.
+* `QUEUE_POLL_INTERVAL`: the queue is polled every minute by default.
+* `REPORTING_FILES_GRAPH`: If a specific graph is needed for the reports (e.g. healing) add URI here.
+* `PUBLISHER_URI (default: "http://data.lblod.info/services/loket-producer")`: URI underneath which delta files will be saved.
+* `KEY (default: '')`: The login key
```

Comment:
in env-config.js, theses seem to be env variables and not from the service config file, e.g:
` this.KEY = process.env.KEY || '';`